geopyspark/geotrellis/render.py

```diff
@@ -49,7 +54,7 @@ def get_hex(geopysc, ramp_name, num_colors=None):
         return list(geopysc._jvm.geopyspark.geotrellis.ColorRamp.getHex(ramp_name))
 
 
-class PngRDD(object):
+class PngRDD(RDDWrapper):
```

Comment:
Is there any benefit for having `PngRDD` being a sublcass of `RDDWrapper`? `PngRDD` has different `persist`, `cache`, and `unpersist` methods, and doesn't get anything from `RDDWrapper`.